### DISCUSSION TITLE
Nhờ

### DISCUSSION THREAD

**Discussion by thongao1302** [hacksider/Deep-Live-Cam#65] [Q&A]
Help me install it

**Comment by nav2924** [comment]
Hey there! Here’s a quick guide to get you started:

Set up your environment:

Make sure you’ve got Python 3.10, pip, git, ffmpeg, and Visual Studio 2022 runtimes (if you’re on Windows).
Clone the Repo:

Run this to grab the project: git clone https://github.com/hacksider/Deep-Live-Cam.git.
Download the Models:

Grab GFPGANv1.4 and inswapper_128_fp16.onnx models.
Drop them into the models folder in the project.
Install Dependencies:

It's best to use a virtual environment to keep things clean.
Install the needed packages with: pip install -r requirements.txt.
Run the Program:

If you don’t have a GPU, just run: python run.py.
Heads up, it might take a bit to download some models the first time.
Optional: GPU Acceleration:

If you want to speed things up with GPU, you’ll need to install the right stuff depending on your hardware (like CUDA for Nvidia).
Swap out onnxruntime for your specific provider using pip uninstall and pip install.
Then run it with: python run.py --execution-provider <your_provider>.
Hope this helps!

**Reply by PrinceSanguan** [reply]
how to fix this? when i run python run.py ">>
Traceback (most recent call last):
  File "C:\Users\Dionisio Samillano\OneDrive\Desktop\Python Projects\Deep-Live-Cam\run.py", line 3, in <module>
    from modules import core
  File "C:\Users\Dionisio Samillano\OneDrive\Desktop\Python Projects\Deep-Live-Cam\modules\core.py", line 14, in <module>
    import torch
  File "C:\Users\Dionisio Samillano\OneDrive\Desktop\Python Projects\Deep-Live-Cam\venv\Lib\site-packages\torch\__init__.py", line 148, in <module>
    raise err
OSError: [WinError 126] The specified module could not be found. Error loading "C:\Users\Dionisio Samillano\OneDrive\Desktop\Python Projects\Deep-Live-Cam\venv\Lib\site-packages\torch\lib\fbgemm.dll" or one of its dependencies."

**Reply by nav2924** [reply]
Type 
pip install torch
pip install core 
in cmd

**Reply by NguyendatGH** [reply]
Can i do it the same on ubuntu 22.4? i'm currently use dualboot.

**Reply by anhdungadg** [reply]
> Can i do it the same on ubuntu 22.4? i'm currently use dualboot.

Yes, you can! In terminal, the best way is using virtual environment in Python: `python -m venv myVenv`

**Reply by mdmestre** [reply]
> Type pip install torch pip install core in cmd

not work

**Comment by anhdungadg** [comment]
> 
> 
> Help me install it

Cài được chưa bạn? Trên Windows cũng khá dễ. 

Have you installed it yet? It's quite easy on Windows OS.
